internal/oci/runtime_oci.go

```diff
@@ -107,7 +99,13 @@ func (r *runtimeOCI) CreateContainer(c *Container, cgroupParent string) (err err
 		"--exit-dir", r.config.ContainerExitsDir,
 		"--socket-dir-path", r.config.ContainerAttachSocketDir,
 		"--log-level", logrus.GetLevel().String(),
-		"--runtime-arg", fmt.Sprintf("%s=%s", rootFlag, r.root))
+		"--runtime-arg", fmt.Sprintf("%s=%s", rootFlag, r.root),
+		"--syslog"}
```

Comment:
syslog used to be added only for non-systemd case. What is the reason to change it? (sorry if I missed it)

Reply:
ok I dropped this and will add in a follow up PR